web/js/map-OpenLayers.js

```diff
@@ -925,7 +935,21 @@ OpenLayers.Control.PermalinkFMS = OpenLayers.Class(OpenLayers.Control.Permalink,
         if ( alter_zoom ) {
             zoom += fixmystreet.zoomOffset;
         }
-        href += separator + OpenLayers.Util.getParameterString(this.createParams(center, zoom));
+
+        var params = this.createParams(center, zoom);
+        // Roundtripping a param containing urlencoded commas through
+        // createParams and getParameterString results in the commas
+        // becoming un-urlencoded, so let's avoid this by joining the arrays
```

Comment:
What's the problem being solved here, why does it matter if they're un-urlencoded? I'm presumably missing something.

Reply:
It's not solving a problem as such, it was to get the frontend tests to pass. Specifically [`cy.url().should('include', 'status=closed%2Cfixed');`](https://github.com/mysociety/fixmystreet/blob/6e2c77b2e1f3f693d806a9c0237403e3273378af/.cypress/cypress/integration/around_filters.js#L22). Not sure if there's a better way, so I just went with consistency.

Reply:
I think we can update the test :) Guess slight concern as to what 'runs' first, perhaps, though? Or can we know that for sure.

Reply:
Ah I think the issue might be the call to updateLink being in `categories_or_status_changed` (called on change and on history navigation) rather than `categories_or_status_changed_history` (only called on actual change) - we don't need to update the URL when clicking back/fwd as it's already done for us. I think?